main/src/cgeo/geocaching/connector/gc/GCMap.java

```diff
@@ -187,6 +187,9 @@ private static void fillForBasicFilter(@NonNull final BaseGeocacheFilter basicFi
                 fillForBasicFilter(((DifficultyAndTerrainGeocacheFilter) basicFilter).difficultyGeocacheFilter, search);
                 fillForBasicFilter(((DifficultyAndTerrainGeocacheFilter) basicFilter).terrainGeocacheFilter, search);
                 break;
+            case RATING:
```

Comment:
You can simply leave out the "case RATING" statement, the "not-supported-case" is handled by "default"

Reply:
Yes, I know. I left it in as explicit documentation that this connector is not able to handle such requests. But I can remove it, if requested.